packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -33,6 +33,7 @@ export class OrgList implements vscode.Disposable {
       49
     );
     this.statusBarItem.command = 'sfdx.force.set.default.org';
+    this.statusBarItem.tooltip = 'Change Default Org';
```

Comment:
Same as above, lets move it to the messages module and use `nls.localize(...)` here